### PR TITLE
feat: Support poller release and pull request periods

### PR DESCRIPTION
**Pull request by dippynark**
This PR allows the period between release and pull request polls to be specified separately.

### CODE REVIEW

File: cmd/poller/main.go [hunk 1]
```diff
@@ -94,6 +95,19 @@ func gatherOptions(fs *flag.FlagSet, args ...string) options {
 	return o
 }
 
+func parseEnvPollPeriod(env string, defaultPollPeriod time.Duration) time.Duration {
+	text := os.Getenv(env)
+	if text != "" {
+		d, err := time.ParseDuration(text)
+		if err != nil {
+			logrus.WithError(err).Warn(fmt.Sprintf("invalid %s value", env))
```
Comment (eddycharly) [inline cmd/poller/main.go:103]: Can we add a more detailed message ?
Actual value and expected format would be great.

Comment (eddycharly) [reply cmd/poller/main.go:103]: Maybe using something like WithField ?

Comment (dippynark) [reply cmd/poller/main.go]: Added some improvements, hopefully that covers it

Comment (eddycharly) [reply cmd/poller/main.go]: Looks good, thanks !

**Pull request merged**